logicrunner/builtin/contract/noderecord/noderecord.go

```diff
@@ -58,16 +58,12 @@ func NewNodeRecord(publicKey string, roleStr string) (*NodeRecord, error) {
 	}, nil
 }
 
-var INSATTR_GetNodeInfo_API = true
-
 // GetNodeInfo returns RecordInfo.
 // ins:immutable
 func (nr NodeRecord) GetNodeInfo() (RecordInfo, error) {
 	return nr.Record, nil
 }
 
-var INSATTR_GetPublicKey_API = true
```

Comment:
INSATTR_GetPublicKey_API this is nesussary for our API. Add nolint if your linter wants to delete it